server/server.js

```diff
@@ -3,7 +3,7 @@ import http from "http";
 import app from "./app";
 import { connectDb, disconnectDb } from "./db";
 
-const port = parseInt(process.env.PORT || "3000");
+const port = parseInt(process.env.PORT || "300");
```

Comment:
Ports below 1000 require administrator privileges so, even though you don't want it to conflict with the default one, you should pick a higher one.